test/tekton-tests.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs the integration tests with Tekton
+
+# In order to run the tests locally against a standalone Kubernetes cluster
+# and container registry, you must set the $CONTAINER_REGISTRY and $DOCKER_CONFIG_JSON
+# environment variables and login to your registry of choice.
+
+source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/e2e-tests.sh
+source $(dirname $0)/e2e-common.sh
+
+# Add local dir to have access to built kn
+export PATH=$PATH:${REPO_ROOT_DIR}
+
+# Script entry point.
+initialize $@
+
+export TEKTON_VERSION=${TEKTON_VERSION:-v0.8.0}
+export KN_E2E_NAMESPACE=tkn-kn
+
+header "Running integration tests for Tekton"
+
+# Install Tekton if not already installed
+if [[ $(kubectl api-resources | grep -c tekton.dev) -eq 0 ]]; then
+  kubectl apply -f https://github.com/tektoncd/pipeline/releases/download/${TEKTON_VERSION}/release.yaml
+fi
+
+if (( IS_PROW )); then
+  # Configure Docker so that we can create a secret for GCR
+  gcloud auth configure-docker
+  gcloud auth print-access-token | docker login -u oauth2accesstoken --password-stdin https://gcr.io
+  export CONTAINER_REGISTRY=gcr.io/${E2E_PROJECT_ID}/${E2E_BASE_NAME}-e2e-img/${RANDOM}
```

Comment:
This will create a separate folder in GCR for your pipeline. So you know, in case you want to keep everything under the same folder, everything else will be using $KO_DOCKER_REPO.

Reply:
Yes. I understand that. Just didn't want to confuse anybody by using the `KO_` prefix because here we're not using `ko`